business/checkers/destination_rules_checker.go

```diff
@@ -10,10 +10,12 @@ import (
 const DestinationRuleCheckerType = "destinationrule"
 
 type DestinationRulesChecker struct {
-	DestinationRules []kubernetes.IstioObject
-	MTLSDetails      kubernetes.MTLSDetails
-	ServiceEntries   []kubernetes.IstioObject
-	Namespaces       []models.Namespace
+	DestinationRules         []kubernetes.IstioObject
+	ExportedDestinationRules []kubernetes.IstioObject
+	MTLSDetails              kubernetes.MTLSDetails
+	ServiceEntries           []kubernetes.IstioObject
+	ExportedServiceEntries   []kubernetes.IstioObject
+	Namespaces               []models.Namespace
```

Comment:
I'd remove this from this branch unless you want to add the validations. It'd reduce dead code and wouldn't confuse us. To me this is a partial implementation. Otherwise, we should add the exported objects to all the other ObjectCheckers (because besides SE, DR and VS, other Object checkers use objects which have exportTo resources).
+1 to remove dead code from `DestinationRulesChecker` and `ServiceEntryChecker`